README.md

```diff
@@ -40,6 +40,26 @@ If you want to make NixOS your default distribution, you can do so with
 wsl -s NixOS
 ```
 
+## Troubleshooting
+
+A recovery shell can be started with
+
+```sh
+wsl -d NixOS --system --user root -- /mnt/wslg/distro/bin/nixos-wsl-recovery
+```
+
+This will load the WSL "system" distribution, activate your configuration,
+then chroot into your NixOS system, similar to what `nixos-enter` would do
+on a normal NixOS install.
+
+You can choose an older generation to load with
```

Comment:
Should we add a subcommand to list available generations?

Reply:
I don't really want to make that wrapper any bigger than it needs to be, but maybe we should document that.

Reply:
Once we have it, yes, though I'd like to also have the fallback for people that install from their own tarballs.